projects/igniteui-angular/schematics/utils/dependency-handler.ts

```diff
@@ -59,16 +59,28 @@ function getTargetedProjectOptions(project: WorkspaceProject<ProjectType>, targe
     throw new SchematicsException(`Cannot determine the project's configuration for: ${target}`);
 }
 
-export function getConfigFile(project: WorkspaceProject<ProjectType>, option: string): string {
-    const buildOptions = getTargetedProjectOptions(project, 'build');
-    if (!buildOptions[option]) {
-        throw new SchematicsException(`Could not find the project ${option} file inside of the ` +
-            `workspace config (${project.sourceRoot})`);
-    }
+export function getConfigFile(project: WorkspaceProject<ProjectType>, option: string, configSection: string): string {
```

Comment:
I'd add `configSection` w/ a default value, (since, up until now, it's always been used for the 'build' section), so that the changes to the `ng-add/index.ts` file won't be needed
```suggestion
export function getConfigFile(project: WorkspaceProject<ProjectType>, option: string, configSection: string = 'build'): string {
```

Reply:
It seems nice, although I like a bit passing the 'build'/'test'  param for readability;
Ref: getConfigFile(project, 'main', 'build');

Reply:
Yeah, you're right, without meaningful `enum`s they're just *some strings* in the moment - leaving them is fine too!